README.md

```diff
@@ -91,6 +83,32 @@ Host Docker storage drivers of `overlay` and `aufs` are supported automatically.
     vagrant ssh
     ```
 
+1. Configure DC/OS Docker:
+
+    Generate `make-config.mk` automatically, interactively, or manually.
+
+    **Automatic Mode:**
+
+    ```console
+    ./configure --auto
+    ```
+
+    **Interactive Mode:**
+
+    ```console
+    ./configure
+    ```
+
+    **Manual Mode (example):**
+
+    ```console
+    cat > make-config.mk << EOM
+    MASTERS := 3
+    EOM
+    ```
+
+    See [make-defaults.mk](make-defaults.mk) for a full list of manually configurable options.
```

Comment:
I refactored the volume variables to clean them up. Now there are volume mount variables for each node type:

```
# Custom Docker volume arguments (passed to docker run)
# Syntax: -v [<host-path>:]<container-path>
CUSTOM_MASTER_VOLUMES :=
CUSTOM_AGENT_VOLUMES :=
CUSTOM_PUBLIC_AGENT_VOLUMES :=
```